Gemfile.lock

```diff
@@ -124,9 +126,6 @@ GEM
       rack (>= 1.0, < 3)
     rainbow (3.0.0)
     rake (10.3.2)
```

Comment:
- update of `rake` is missing and other gems
- you should use `remote_ip` in addition : https://github.com/Mapotempo/geocoder-api/commit/0b2eafe588399b30f976064bb48704d26d9e84e1